docs/guides/platform/get-started/getting-started-with-linode/index.md

```diff
@@ -61,10 +62,14 @@ Use a [StackScript](http://www.linode.com/stackscripts) to quickly deploy softwa
 
 1. Select a Linode plan.
 
+![Getting Started Create Linode](getting-started-create1.png "Getting Started Create Linode")
+
 1.  Give your Linode a label. This is a name to help you easily identify it within the Cloud Manager's Dashboard. If desired, assign a tag to the Linode in the **Add Tags** field.
 
 1. Create a root password for your Linode in the **Root Password** field. This password must be provided when you log in to your Linode via SSH. The password must meet the complexity strength validation requirements for a strong password. Your root password can be used to perform any action on your server, so make it long, complex, and unique.
 
+![Getting Started Create Linode 2](getting-started-create2.png "Getting Started Create Linode 2")
```

Comment:
Please indent the image because the steps are getting renumbered:https://deploy-preview-3873--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/getting-started/